contracts/lbp/LBPWrapper.sol

```diff
@@ -167,6 +167,69 @@ contract LBPWrapper {
         vault.joinPool(lbp.getPoolId(), address(this), address(this), request);
     }
 
+    /**
+     * @dev exit pool or remove liquidity from pool
+     * @param _tokens                   array of tokens sorted for the LBP
+     * @param _receiver                 address who will be credited with the tokens after removing liquidity
+     * @param _minAmountsOut            array of minimum amounts out
+     * @param _toInternalBalance        fund tokens to the internal user balance
+     * @param _userData                 userData specifies the type of exit
+     */
+    function removeLiquidity(
+        IERC20[] memory _tokens,
+        address payable _receiver,
+        uint256[] memory _minAmountsOut,
+        bool _toInternalBalance,
+        bytes memory _userData
+    ) external {
+        require(
+            _receiver != payable(address(0)),
+            "LBPWrapper: receiver of project and funding tokens can't be zero"
+        );
+        require(holdsPoolTokens(), "LBPWrapper: pool tokens were withdrawn");
+        require(endTimeReached(), "LBPWrapper: cannot exit before endtime");
+
+        IVault vault = lbp.getVault();
+
+        IVault.ExitPoolRequest memory request = IVault.ExitPoolRequest({
+            minAmountsOut: _minAmountsOut,
```

Comment:
Do they have the option here to only withdraw part of the liquidity? Do we want to give this option to them?

Reply:
hm, we can raise this question in today's standup.

Reply:
I thought this is something that was already discussed. Maybe lets check the miro board, just to make sure if ther's not a notation anywhere there before asking.